Assets/Mirror/Runtime/ServerObjectManager.cs

```diff
@@ -430,6 +444,7 @@ internal void SpawnObject(GameObject obj, INetworkConnection ownerConnection)
                 // the object has not been spawned yet
                 identity.NetId = GetNextNetworkId();
                 Server.Spawned[identity.NetId] = identity;
+                Spawned.Invoke(identity);
```

Comment:
This is before `identity.StartServer` but the client side one is after `identity.StartClient`, is probably worth calling the server one after to so it is consistent. 

I think there will need to be more document to explain the order of execution to make these easier to use.

Reply:
good call